### DISCUSSION TITLE
[Coding Habits] Would be nice if we have a max. num of language activity option for this plugin

### DISCUSSION THREAD

**Discussion by billpwchan** [lowlighter/metrics#1064] [Ideas]
Hi lowlighter,

For the Coding Habits plugin, from what I understand is that it tries to collect the languages used in the recent X events (1 < X < 1000), but we do not have an option to control the number of languages displayed similar to the one in the Languages plugin. Thus, the following thing happened when I tried to add some random languages to my repository lol

**Would be great to have an option to control this! Let me know what's your thought on this idea😊**
![image](https://user-images.githubusercontent.com/25129943/169578764-50a77f22-cc81-4500-a77e-5cd7a064bf2c.png)

**Comment by lowlighter** [comment]
Yeah indeed 😅

**Reply by lowlighter** [reply]
Was shipped in #1112 and #1138
